modules/ROOT/pages/migrate-cl.adoc

```diff
@@ -1,49 +1,51 @@
+* flannel is not supported on FCOS.
+* The functionality of the reboot manager (`locksmith`) is rolled into https://github.com/coreos/zincati/blob/master/README.md[Zincati].
+* Be default, FCOS does not allow SSH password logins. For full details on how to set up users, refer to the documentation for the xref:ign-passwd.adoc[`passwd`] node of the FCC file.
+* FCOS does not have a mechanism to select the version of docker.
+* Because `usermod` is not yet fully-functional on FCOS, there is a `docker` group in the `/etc/group` file. This is a stop-gap measure to facilitate a smooth transition to FCOS. The team is working on a more functional `usermod`, at which time the `docker` group will no longer be included by default. See the https://github.com/coreos/fedora-coreos-tracker/issues/2[docker group issue].
+* There is no way to create directories below the `/` directory. Changes are restricted to `/etc` and `/var`. Refer to the documentation for the xref:ign-storage.adoc[`storage`] node of the FCC configuration for details about writing directories and files to FCOS.
 
 == Operator notes
-* CL had three release channels: Stable, Beta, and Alpha. FCOS maintains the Fedora standard of two release streams: test and stable. See the https://getfedora.org/coreos/download/[download page] for more details.
-* Automatic updates are coordinated by Zincati, as described in the https://github.com/coreos/zincati/blob/master/docs/usage/auto-updates.md[Zincati documentation]. Each update requires a system reboot, and https://github.com/coreos/rpm-ostree/blob/master/README.md[`rpm-ostree`] provides a rollback mechanism in the grub menu.
-* The `update-ssh-keys` tool is not shipped on FCOS. The preferred method is to use the https://www.ssh.com/ssh/authorized_keys/openssh/[`AuthorizedKeysFile`].
-//*SELinux policy differences. What are they?
-* In CL, metrics/telemetry data was collected by the update mechanism. In FCOS, the plan is to collect data (without unique identifiers) via https://github.com/coreos/fedora-coreos-pinger/issues/30[`fedora-coreos-pinger`]. Users will have the ability to disable data collection.
+* CL had three release channels: Stable, Beta, and Alpha. The FCOS production https://github.com/coreos/fedora-coreos-tracker/blob/master/Design.md#release-streams[release streams] are `testing`, `stable`, and `next`.
+* Automatic updates are now coordinated by Zincati, as described in the https://github.com/coreos/zincati/blob/master/docs/usage/auto-updates.md[Zincati documentation]. The rollback mechanism (via grub) is now provided by https://github.com/coreos/rpm-ostree/blob/master/README.md[`rpm-ostree`].
+* The `update-ssh-keys` tool is not shipped on FCOS. To update ssh keys, edit `authorized_keys` directly. Keep in mind that Ignition and Afterburn have their own files in `~/.ssh/authorized_keys.d`.
```

Comment:
"edit `~/.ssh/authorized_keys` directly"
"Keep in mind" isn't the right phrasing, since users wouldn't be expected to know that.  `Note that` would be better.